stable/admin/README.md

```diff
@@ -182,10 +182,13 @@ The following tables list the configurable parameters for the `admin` option of
 | `tolerations` | Tolerations for NuoDB Admin | `[]` |
 | `configFilesPath` | Directory path where `configFiles.*` are found | `/etc/nuodb/` |
 | `configFiles.*` | See below. | `{}` |
-| `persistence.enabled` | Whether or not persistent storage is enabled for admin RAFT state | `false` |
```

Comment:
so RAFT now has to be persistent in all cases?

Reply:
i removed this because (I think) we had removed the actual option to disable some time ago - I can't see any references to admin.persistence.enabled...